### DISCUSSION TITLE
Mouse Sensitivity

### DISCUSSION THREAD

**Discussion by Fikrienz** [ppy/osu#28249] [Q&A] [answered]
I'm playing osu! Lazer on mobile with OTG. After this update i didn't see the mouse sensitivity option.
![Screenshot_2024-05-20-12-05-04-38_25a1a32208bbcdc1d450b7aa854bb161](https://github.com/ppy/osu/assets/157803695/d3e323cc-2cea-4183-9bca-b4b1395def7d)
 Is it a bug or the update part?

**Comment by uzervlad** [comment, marked as answer]
#28226 #28228
